README.md

```diff
@@ -8,6 +8,8 @@ The plugin includes two builders which are able to create images, depending on y
 
 * `vsphere-clone` - This builder clones an existing virtual machine template, modifies the template, and then saves it as a new image. The builder uses the vSphere API to build images on a vSphere cluster or ESXi host by connecting to a vCenter Server instance.
 
+* `vsphere-supervisor` - This builder deploys a vm to a vSphere Supervisor cluster by using the VM-Service API.
```

Comment:
```suggestion
* `vsphere-supervisor` - This builder creates a virtual machine on  a vSphere Supervisor cluster by using the VM-Service API.
```

Reply:
This may be more accurate. May need an update after adding the publish capabilities.